v2/internal/controller/controllers/generic_controller.go

```diff
@@ -179,6 +183,10 @@ func MakeResourceGVKLookup(mgr ctrl.Manager, objs []client.Object) (map[schema.G
 	return result, nil
 }
 
+// NamespaceAnnotation defines the annotation name to use when marking
+// a resource with the namespace of the managing operator.
+const NamespaceAnnotation = "azure.microsoft.com/operator-namespace"
```

Comment:
I think they're naturally disjoint because of the different API groups, right? I can't see that we'd have v1 and v2 operators both trying to manage the same resources.

Reply:
Yeah, I'm silly. I read this as "an annotation on the namespace" which I know is 100% not what it is. Ignore this =)